### ISSUE TITLE
screen log

### ISSUE THREAD

**Issue by FabrizioLacalandra** [JuliaOpt/Xpress.jl#3]
Is there a way to have a screen log? In the optimizer manual it is said that OUTPUTLOG controls the stdout, but "However, under Windows, no output from the Optimizer DLL is sent to the screen. The user must define a callback function and print messages to the screen them self if they wish output to be displayed". But i think callback have not been implemented yet in Xpress.jl. or there is some other way ?

Apart from that the other param could be send to the solver  as for the other solvers e.g. 
Model(solver = XpressSolver(MIPRELSTOP=0.1))
right ?

Thanks 
Fabrizio

**Comment by joaquimg** [comment]
Unfortunately I could not figure out a way to print the log to screen on windows.

The callbacks have no been implemented also, because they work a bit different from most other solver. It is possible to get them working, but I havent had the time.

Yes, you can send params just like the other solvers.

**Comment by joaquimg** [comment]
One more thing,

It is possible to wrap the function XPRSsetlogfile, and its very easy, since it only takes to parameters: the model pointer and the filename.

That way you can still get the log somewhere. If I remeber correctly my last use, you can open the file while its still being written by xpress.

**Comment by joaquimg** [comment]
You can use `Xpress.setlogfile(m, filename)` that will be available in version 0.0.3, hopefully by tomorrow

its is simply:
```
function setlogfile(model::Model, filename::Compat.ASCIIString)
    #int XPRS_CC XPRSsetlogfile(XPRSprob prob, const char *filename);

    flags = ""
    ret = @xprs_ccall(setlogfile, Cint, (Ptr{Void}, Ptr{UInt8}),
        model.ptr_model, filename)
    if ret != 0
        throw(XpressError(model))
    end
    nothing
end
```
